lib/ide/ide/Cargo.toml

```diff
@@ -0,0 +1,48 @@
+[package]
+name = "ide"
+version = "0.1.0"
+authors = ["Enso Team <contact@luna-lang.org>"]
+edition = "2018"
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[dependencies]
+basegl              = { version = "0.1.0"  , path = "../../core" }
+enso-prelude        = { version = "0.1.0"  , path = "../../prelude" }
+file-manager-client = { version = "0.1.0"  , path = "../file-manager" }
+json-rpc            = { version = "0.1.0"  , path = "../json-rpc" }
+utils               = { version = "0.1.0"  , path = "../utils" }
+
+serde = { version = "1.0", features = ["derive"] }
+uuid  = { version = "0.8", features = ["serde", "v5"] }
+
+console_error_panic_hook = "0.1.6"
+failure      = "0.1.6"
+futures      = "0.3.1"
+js-sys       = "0.3.35"
+paste        = "0.1.6"
+serde_json   = "1.0"
+shrinkwraprs = "0.3.0"
+wasm-bindgen = "0.2.58"
+wasm-bindgen-test = "0.3.8"
+
+[target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+websocket = "0.23.0"
+
+
+[dependencies.web-sys]
+version = "0.3.22"
+features = [
+  'Blob',
+  'console',
+  'CloseEvent',
+  'Document',
+  'Element',
+  "ErrorEvent",
+  "MessageEvent",
+  'HtmlElement',
+  'Node',
+  'WebSocket',
+  'Window',
+]
```

Comment:
I wonder if these changes does not go to conflict with @notdanilo work. Please make sure you are working on the same ide lib.

Reply:
It doesn't conflict. I am actually merging this branch into my working branch :)